Practical_Training/Common/Configuration/ExceptionHandlerConfig.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Practical_Training.Models.Share;
+using Microsoft.AspNetCore.Diagnostics;
+using System.Net;
+
+namespace Practical_Training.Common.Configuration
+{
+    public static class ExceptionHandlerConfig
```

Comment:
how does this work? when it will be called?

Reply:
I replace it with CustomExceptionMiddleware more better